### ISSUE TITLE
Using a preset overrides format option

### ISSUE THREAD

**Issue by AxelTerizaki** [AndiDittrich/Node.CLI-Progress#37]
Hello :)

I'm migrating from cliProgress 2.x to 3.0.0 and I'm having an issue.

Here's how I initialized my bar

```
this.format = `${options.message} [{bar}] {percentage}%`;
this.bar = new cliProgress.Bar({
  format: this.format,
  stopOnComplete: true
}, cliProgress.Presets.rect);
```

When I migrated to 3.0.0 I realized my format options was ignored : 

```
 ■■■■■■■■■■■■■■■                         ■ 39% | ETA: 4s | 7472/19357
```

There was ETA and the value/total displayed.

I tried with other presets but it seems to cause the same issue.

At last I tried without any preset and there my format message was used : 

```
Checking files...     [==================----------------------] 44%
```

So I think using a preset mistakenly overrides the format option.

Am I doing it wrong or is it a bug?

Thanks for the multi bar support, I'll need to think of how to use it properly but it's a neat feature :)

**Comment by AndiDittrich** [comment]
it's my fault... the options should take precedence over the preset

**Comment by AndiDittrich** [comment]
fixed within master branch

**Comment by AxelTerizaki** [comment]
Thanks for the quick fix ! I'll have to wait for 3.0.1 to be published on npm I guess :)

**Comment by AndiDittrich** [comment]
the next release will be 3.1.0 including the notty mode
